src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  border: none;
+}
+
+.card {
+  background: #fff;
+  border-radius: 5px;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  font-family: Roboto, sans-serif ;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+    margin-left: 3px;
+    margin-right: 3px;
+  }
+
+  &__title {
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+}
+
+.card__product {
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
```

Comment:
according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 11 there should be 1 file for each BEM block - please create 1 for card and 1 for stars, and move your code there